src/openforms/forms/models/form_statistics.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.utils.formats import localize
+from django.utils.translation import gettext_lazy as _
+
+
+class FormStatistics(models.Model):
+    form = models.ForeignKey(
+        verbose_name=_("form"),
+        to="forms.Form",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+    form_name = models.CharField(
+        verbose_name=_("form name"),
+        max_length=150,
+        help_text=_(
+            "The name of the submitted form. This is saved separately in case of form deletion."
+        ),
+    )
+    submission_count = models.PositiveIntegerField(
+        verbose_name=_("Submission count"),
+        default=0,
+        help_text=_("The number of the submitted forms."),
+    )
+    first_submission = models.DateTimeField(
+        verbose_name=_("first submission"),
+        help_text=_("Date and time of the first submitted form."),
+        auto_now_add=True,
+    )
+    last_submission = models.DateTimeField(
+        verbose_name=_("last submission"),
+        help_text=_("Date and time of the last submitted form."),
+    )
+
+    class Meta:
+        verbose_name = _("Form statistics")
+        verbose_name_plural = _("Form statistics")
```

Comment:
```suggestion
        verbose_name = _("form statistics")
        verbose_name_plural = _("form statistics")
```
you typically want to provide `verbose_name` (also for fields) lower-cased, Django itself takes care of capitalizing it where relevant.